### ISSUE TITLE
[Script] Ensure that constants in Audio/AudioDef.h are being picked up by bindings

### ISSUE THREAD

**Issue by JoshEngebretson** [AtomicGameEngine/AtomicGameEngine#1152]
Not seeing these in script: 
```
/// SoundSource type defaults
static const String SOUND_MASTER = "Master";
static const String SOUND_EFFECT = "Effect";
static const String SOUND_AMBIENT = "Ambient";
static const String SOUND_VOICE = "Voice";
static const String SOUND_MUSIC = "Music";
```

**Comment by JimMarlowe** [comment]
In testing this, I'm finding that the AtomicEditor autocompletion system is picking up the SOUND_XXX entries, and is able to plant them in the code. However, monodevelop does not show any constants as far as I can tell, even the ones that were there before ( for example STREAM_BUFFER_LENGTH ). and the c# feature example for sound effects uses the camel case versions of the sound types, not the upper case ones. Does VS do autocompletion for the sound types and this is a monodevelop issue?

**Comment by JimMarlowe** [comment]
Nevermind, monodevelop does recognize them, they are in the AtomicEngine.Constants class. Not as intuitive as the JS/TS organization.

**Comment by JoshEngebretson** [comment]
Closed via #1183
